spec/models/basic_project_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe BasicProject, type: :model do
```

Comment:
Do I need to add any additional tests for BasicProject?

Reply:
No this is fine, no major logic additions in this issue. Nice to have a basic projects spec file now!